.travis.yml

```diff
@@ -21,6 +21,9 @@ before_install:
   - cd coalahtml/_coalahtml
   - bower install
   - cd -
+  # colorama needs to be upgraded to fix error when installed colorama 
+  # version and required are different.
+  - sudo pip3 install -U colorama
```

Comment:
If some packages need colorama version ~=3.6 and here you'll upgrade to version 4.0. Can this happen?